compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirMemberPropertyChecker.kt

```diff
@@ -40,7 +148,7 @@ object FirMemberPropertyChecker : FirRegularClassChecker() {
         // So, our source of truth should be the full modifier list retrieved from the source.
         val modifierList = with(FirModifierList) { property.source.getModifierList() }
 
-        checkProperty(containingDeclaration, property, modifierList, reporter, context)
+        checkProperty(containingDeclaration, property, modifierList, isInitialized, reporter, context)
```

Comment:
A lot of functions with name `checkProperty` is confusing (especially after removing usage of `modifierList` in that one, because it also redundant). Could you please rename them somehow (I think it's better to rename utulity one rather than private member of `FirMemberPropertyChecker`)

Reply:
This can be fixed as separate PR or as part of some other PR

Reply:
Yeah, let me take a look at them.

Reply:
Sent out https://github.com/JetBrains/kotlin/pull/4178